database/seeders/SlideSeeder.php

```diff
@@ -9,6 +9,9 @@ class SlideSeeder extends Seeder
 {
     public function run()
     {
+        // XXX Without these slides already seeded, this update is moot.
+        // Even if slides are necessary to run the app, this seems like good
+        // functionality to test.
```

Comment:
Yes, this was one-time FE formatting fix when we converted this field from a plain text field to a WYSIWYG editor. I don't think this seeder is needed as it is, but it would be good to test that `end_credit_copy` (and perhaps other WYSIWYG fields?) is wrapped in a `<p>` tag.